os-downstream-tag.py

```diff
@@ -48,17 +48,17 @@ def add_tag(tag, ref):
     return subprocess.check_output(cmd).strip()
 
 
-def push(remote, branch):
-    print("Pushing branch", branch, "to remote", remote)
-    cmd = ["git", "push", remote, branch]
+def push_tag(remote, tag):
+    print("Pushing tag", tag, "to remote", remote)
+    cmd = ["git", "push", remote, tag]
     output = subprocess.check_output(cmd)
     return output.decode(encoding=sys.stdout.encoding).strip()
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("-d", "--downstream-remote", default="stackhpc",
-                        help="Name of the upstream git remote. Default is "
+                        help="Name of the downstream git remote. Default is "
                              "'stackhpc'")
```

Comment:
We talked on Wednesday about making the upstream remote optional, because we only want to consider tags on the downstream branch.

Reply:
As discussed, I misunderstood the use of the upstream branch here. This is fine.